torch/distributions/multivariate_normal.py

```diff
@@ -184,9 +184,11 @@ def covariance_matrix(self):
 
     @lazy_property
     def precision_matrix(self):
-        identity = torch.eye(self.loc.size(-1), device=self.loc.device, dtype=self.loc.dtype)
+        # identity = torch.eye(self.loc.size(-1), device=self.loc.device, dtype=self.loc.dtype)
```

Comment:
Let us remove these commented lines if the `cholesky_inverse` is working correctly.

Reply:
Okay, removed it. Thanks for checking.